tests/lib/prepare-restore.sh

```diff
@@ -492,13 +492,21 @@ prepare_project() {
         rm -rf "${GOPATH%%:*}/src/github.com/kardianos/govendor"
         go get -u github.com/kardianos/govendor
     fi
-    # Retry govendor sync to minimize the number of connection errors during the sync
-    for _ in $(seq 10); do
-        if quiet govendor sync; then
-            break
-        fi
-        sleep 1
-    done
+
+    if [[ "$SPREAD_SYSTEM" == opensuse-tumbleweed-* ]]; then
+        wget -q "https://storage.googleapis.com/spread-snapd-tests/dependencies/govendor/vendor.zip"
```

Comment:
This means we need this cron job to be running. An alternative approach to consider would be to run govendor sync as part of the test workflow on opensuse, and then in the spread job stage run `REPACK_KEEP_VENDOR=y spread -abend ...`. This way we would not need the cron job and storage.

I'm bit worried that we still may end up getting some weird fs errors, even if govendor is not the process that triggers them.